src/livery-bridge-interactive/LiveryBridgeInteractive.ts

```diff
@@ -263,6 +287,30 @@ export class LiveryBridgeInteractive extends LitElement {
               </td>
               <td id="getStreamIdOutput"></td>
             </tr>
+            <tr>
```

Comment:
All InteractiveBridge commands should be accessible through this element. I.e: Add pause, play etc. here. For those commands that require arguments we'll also need to support some inputs with parsing etc. or perhaps just start with hardcoded mock values for now.

See also the mention of perhaps organizing a meeting to discuss the scope of this ticket. I think this component basically provides a part of our testing implementation and thus should provide reasonably complete testing coverage.